author/ref/document-attributes.adoc

```diff
@@ -82,6 +82,9 @@ Two-letter code (ISO 639-1) of the language the document is written in. Defaults
 The script of the document (ISO 15924). Defaults to `Latn`. Must be supplied as
 `Hans` for Simplified Chinese.
 
+`:boilerplate-authority:`::
+File containing boilerplate of document, in Metanorma XML [added in https://github.com/metanorma/metanorma-standoc/releases/tag/v1.3.15].
```

Comment:
Can you help explain how this is to be used? Thanks.

Reply:
The explanation is in a separate branch, so I am merging this branch into that branch.